scripts/diff_stix.py

```diff
@@ -420,19 +429,70 @@ def placard(item):
+                                index = added[0]
+                                value = added[1]
+                                changes.append(f"{indent}* {index}\n")
+                                changes.append(f"{indent}```\n{indent}{value}\n{indent}```\n")
+                        elif change_type == "change":
+                            if index_changed is None:
+                                changes.append(f"{indent}* Changed {what_changed} field")
+                            else:
+                                changes.append(f"{indent}* Changed index {index_changed} in {what_changed} field")
+
+                            indent = "\t" * (tab_count + 1)
+                            old_change = how_changed[0].replace("\n", f"\n{indent}")
+                            new_change = how_changed[1].replace("\n", f"\n{indent}")
+                            changes.append(f"{indent}* Old:  \n{indent}```\n{indent}{old_change}\n{indent}```\n")
+                            changes.append(f"{indent}* New:  \n{indent}```\n{indent}{new_change}\n{indent}```\n")
+                        elif change_type == "remove":
+                            changes.append(f"{indent}* Removed entries from {what_changed} field")
+                            indent = "\t" * (tab_count + 1)
+                            for removed in how_changed:
+                                index = removed[0]
+                                value = removed[1]
+                                removed = f"{index}:  {value}"
+                                changes.append(f"{indent}* {removed}\n")
+                        else:
+                            print(f"Warning:  Unprocessed change:  {change}")
+
+                return "\n".join(changes)
+
+
             # build sectionList string
             sectionString = ""
             for grouping in groupings:
                 if grouping["parentInSection"]:
                     sectionString += f"* { placard(grouping['parent']) }\n"
+                    if section == "changes":
+                        sectionString += f"{change_details_placard(grouping['parent'], 1) }\n"
                 # else:
                 #     sectionString += f"* _{grouping['parent']['name']}_\n"
                 for child in sorted(grouping["children"], key=lambda child: child["name"]):
                     if grouping["parentInSection"]:
                         sectionString += f"\t* {placard(child) }\n"
+                        if section == "changes":
+                            sectionString += f"{change_details_placard(child) }\n"
                     else:
                         sectionString += f"* { grouping['parent']['name'] }: { placard(child) }\n"
+                        if section == "changes":
+                            sectionString += f"{change_details_placard(child, 1) }\n"
 
+            print(f"Number of (sub-)technique changes in section {section} is {self.change_details['num_changes']}")
```

Comment:
This also makes more sense as line at the end of the output change section (`sectionString`), since seeing it in the output print isn't that useful. It also interrupts the "generating markdown string... done" output when `--verbose` is specified:

```
generating markdown string... Number of (sub-)technique changes in section additions is 0
Number of (sub-)technique changes in section additions is 0
Number of (sub-)technique changes in section minor_changes is 0
Number of (sub-)technique changes in section deletions is 0
done
```